content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md

```diff
@@ -12,9 +12,34 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-Istio supports proxying all TCP traffic by default, but in order to provide additional capabilities,
-such as routing and rich metrics, the protocol must be determined.
-This can be done automatically or explicitly specified.
+Istio supports proxying any TCP traffic. This includes HTTP, HTTPS, gRPC, as well as raw TCP protocols.
+In order to provide additional capabilities, such as routing and rich metrics, the protocol must be determined. This can be done automatically or explicitly specified.
+
+Non-TCP based protocols, such as UDP, are not proxied. These protocols will continue to function as normal, without
+any interception by the Istio proxy. They are not supported at gateways.
```

Comment:
"They are not supported at gameways." is a little vague. Can you clarify? Would something like this be correct?
```suggestion
any interception by the Istio proxy but cannot be used in proxy-only components such as ingress or egress gateways.
```

Reply:
I think that makes sense. Basically the idea is UDP is not "supported" its just not touched at all. This means you cannot use it at gateway, but in a pod it can send UDP out untouched.